demos/can-component/selected.html

```diff
@@ -12,7 +12,7 @@
 <script type="text/stache" id="demo-html">
 {{#items}}
   <div {{#isSelected()}}class='selected'{{/isSelected}}
```

Comment:
I don't think this will work (in 4.0)... `isSelected()` won't be passed an `options` argument. You should just pass the item in like `isSelected(this)`